tests/acceptance/NewsletterStatisticsCest.php

```diff
@@ -20,10 +20,18 @@
 
 class NewsletterStatisticsCest {
 
+  /** @var Settings */
+  private $settings;
+
+  protected function _inject(Settings $settings) {
+    $this->settings = $settings;
+  }
+
   function _before(\AcceptanceTester $I) {
     $I->activateWooCommerce();
-    (new Settings())->withWooCommerceListImportPageDisplayed(true);
-    (new Settings())->withDisplayRevenuesEnabled();
+    $this->settings->withWooCommerceListImportPageDisplayed(true);
+    $this->settings->withDisplayRevenuesEnabled();
+    $this->settings->withCookieRevenueTrackingDisabled();
```

Comment:
I think that we will need to add this setting to more tests (probably all which work with WooCommerce), but it could be probably done once we remove the feature switch or turn the feature on by default.